src/app/forms/controls/select.html

```diff
@@ -0,0 +1,123 @@
+<!--
+  ~ Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+  ~ This software is released under MIT license.
+  ~ The full license information can be found in LICENSE in the root directory of this project.
+  -->
+
+<h2>Select</h2>
+
+<form class="clr-form">
+    <div class="clr-form-control">
+        <label class="clr-control-label">
+            Basic
+        </label>
+        <div class="clr-control-container">
+            <div class="clr-select-wrapper">
+                <div class="clr-select">
```

Comment:
Do we need both `.clr-select-wrapper` and `.clr-select`? The error Icon can go inside of the `.clr-select` div after the `select`, no?

Reply:
This was the only way to keep the `.clr-select` element from having sizing issues from other elements in the flex layouts.

Reply:
As we reviewed, we got this to one level now :)